src/content/docs/Generic Programming/compiletime.md

```diff
@@ -275,32 +296,68 @@ $assert($arg > 3);
 
 ### `$assignable`
 
-Check if an expression is assignable to the given type, e.g. `Type x = expr;` would be valid.
+`$assignable(<expr>, <Type>)` checks whether an expression is assignable to the given type, meaning that `Type id = expr;` would be valid for some identifier `id`. 
+
+For example, `$assignable("7", int)` tests whether the string `"7"` can be assigned to an `int`. It can't. This isn't JavaScript, thankfully. Such tests are useful for generic code and testing the operating environment, so that modules and macros can behave correctly or at least report errors when incompatibilities are present.
```

Comment:
Maybe remove the reference to JS. But more importantly, this is largely superseded by the macro `@assignable_to` which uses `$define` under the covers. So this part should be removed.